src/services/dashboards/dashboard-customize/DashboardCustomizePage.vue

```diff
@@ -20,48 +20,91 @@
                                         refresh-disabled
             />
         </div>
+        <dashboard-widget-container
+            ref="widgetContainerRef"
+            :dashboard-widget-layouts="state.dashboardWidgetLayouts"
+            :loading.sync="state.loading"
+        />
         <dashboard-customize-sidebar />
         <dashboard-manage-variable-overlay :visible="variableState.showOverlay" />
     </div>
 </template>
 
 <script setup lang="ts">
 import type Vue from 'vue';
-import { computed, getCurrentInstance, reactive } from 'vue';
+import {
+    computed, getCurrentInstance, onMounted, reactive,
+} from 'vue';
 
 import { PButton, PDivider } from '@spaceone/design-system';
 import dayjs from 'dayjs';
 
 import { SpaceRouter } from '@/router';
 
+import ErrorHandler from '@/common/composables/error/errorHandler';
+
 import type { DateRange } from '@/services/dashboards/config';
 import DashboardManageVariableOverlay
     from '@/services/dashboards/dashboard-customize/modules/dashboard-manage-variable-overlay/DashboardManageVariableOverlay.vue';
 import DashboardCustomizeSidebar from '@/services/dashboards/dashboard-customize/modules/DashboardCustomizeSidebar.vue';
+import DashboardWidgetContainer from '@/services/dashboards/dashboard-detail/modules/DashboardWidgetContainer.vue';
+import { DASHBOARD_TEMPLATES } from '@/services/dashboards/default-dashboard/template-list';
+import type { DashboardModel } from '@/services/dashboards/model';
 import DashboardLabels from '@/services/dashboards/modules/dashboard-label/DashboardLabels.vue';
 import DashboardToolset from '@/services/dashboards/modules/dashboard-toolset/DashboardToolset.vue';
 import DashboardRefreshDropdown from '@/services/dashboards/modules/DashboardRefreshDropdown.vue';
+import type { DashboardLayoutWidgetInfo } from '@/services/dashboards/widgets/config';
+
+interface Props {
+    dashboardId: string;
+}
 
 const MANAGE_VARIABLES_HASH_NAME = 'manage-variables';
 
+const props = defineProps<Props>();
 const state = reactive({
     labelList: [] as Array<string>,
     refreshInterval: undefined,
     dateRange: {
         start: dayjs.utc().format('YYYY-MM-01'),
         end: dayjs.utc().format('YYYY-MM-DD'),
     } as DateRange,
+    isProjectDashboard: computed<boolean>(() => Boolean(props.dashboardId.startsWith('project'))),
+    dashboardInfo: {} as DashboardModel,
+    dashboardWidgetLayouts: computed<DashboardLayoutWidgetInfo[][]>(() => state.dashboardInfo?.layouts ?? []),
```

Comment:
Just question: What is the difference between using `startsWith` alone and using `Boolean()`?

Reply:
Oh I just copied @minseolee 's code 😅
I think there's no difference.

Reply:
Oops, my mistake 
I'll apply it!